thinc/backends/numpy_ops.pyx

```diff
@@ -616,7 +611,7 @@ cdef void cpu_update_averages(weight_t* ema,
         ema[i] -= one_minus_decay * (ema[i] - weights[i])
 
 
-def lstm_forward_training(
+def lstm_forward_training(CBlas cblas,
```

Comment:
Why not use `self.cblas()` like `gather_add` does? It's a little more indirect but doesn't require changing the function args. Or is there a reason you want to be able to pass a specific (different) `Cblas`?

Reply:
This function isn't a method of `Ops`, so that's not an option here. As for the above methods, we could very well call `self.cblas()` instead of instantiating the `CBlas` struct directly. We save a Python call with the latter, but the overhead will be negligible in most, if not all, cases. I can change that.